core/unit_test/TestTeamMDRange.hpp

```diff
@@ -923,7 +1049,13 @@ struct TestTeamThreadMDRangeParallelReduce : public TestTeamMDParallelReduce {
                 threadSum += v(leagueRank, i, j);
               },
               teamSum);
-          leagueSum += teamSum;
+      // FIXME_OPENMPTARGET
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+          if (team.team_rank() == 0) leagueSum += teamSum;
+#else
+          Kokkos::single(Kokkos::PerTeam(team),
+                         [&]() { leagueSum += teamSum; });
+#endif
```

Comment:
Could you remind me why `Kokkos::single` with `PerTeam` can't be used for `OMPTarget` here?

Reply:
It failed to compile.